server/src/main/java/org/apache/uniffle/server/HealthCheck.java

```diff
@@ -38,14 +39,15 @@ public class HealthCheck {
 
   private static final Logger LOG = LoggerFactory.getLogger(HealthCheck.class);
 
-  private final AtomicBoolean isHealthy;
+  private AtomicReference<ServerStatus> serverStatus;
   private final long checkIntervalMs;
   private final Thread thread;
   private volatile boolean isStop = false;
   private List<Checker> checkers = Lists.newArrayList();
 
-  public HealthCheck(AtomicBoolean isHealthy, ShuffleServerConf conf, List<Checker> buildInCheckers) {
-    this.isHealthy = isHealthy;
+  public HealthCheck(AtomicReference<ServerStatus> serverStatus,
```

Comment:
Could we use style consistent with other places?
```
public HealthCheck(
       AtomicReference<ServerStatus> serverStatus,
       ShuffleServerConf conf,
      List<Checker> buildInCheckers) {
```

Reply:
@yl09099  Could you address this comment?

Reply:
This indent should be 4.

Reply:
sorry, Have been changed